src/main/java/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoFactory {
+
+    private LottoFactory() {
+
+    }
```

Comment:
요런 경우 한 줄로 정리해주시는 것이 보기에 좋습니다.

```suggestion
    private LottoFactory () {}